src/main/java/uk/gov/hmcts/reform/ccd/document/am/feign/CaseDocumentClient.java

```diff
@@ -0,0 +1,101 @@
+package uk.gov.hmcts.reform.ccd.document.am.feign;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.ccd.document.am.model.Document;
+import uk.gov.hmcts.reform.ccd.document.am.model.DocumentTTLRequest;
+import uk.gov.hmcts.reform.ccd.document.am.model.DocumentTTLResponse;
+import uk.gov.hmcts.reform.ccd.document.am.model.DocumentUploadRequest;
+import uk.gov.hmcts.reform.ccd.document.am.model.Classification;
+import uk.gov.hmcts.reform.ccd.document.am.model.UploadResponse;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.requireNonNull;
+
+@Service
+public class CaseDocumentClient {
+
+    private CaseDocumentClientApi caseDocumentClientApi;
+
+    @Autowired
+    public CaseDocumentClient(CaseDocumentClientApi caseDocumentClientApi) {
+        this.caseDocumentClientApi = caseDocumentClientApi;
+    }
+
+    public UploadResponse uploadDocuments(String authorisation, String serviceAuth, String caseTypeId,
+                                          String jurisdictionId, List<MultipartFile> files) {
+        return uploadDocuments(authorisation, serviceAuth, caseTypeId, jurisdictionId, files,
+                               Classification.RESTRICTED);
+    }
+
+    private UploadResponse uploadDocuments(String authorisation, String serviceAuth,
```

Comment:
Do we need this method - could put the implementation into the method defined on line 36 directly, as nothing else calls it?

Reply:
For this one instead of deleting ive made it public so the client can call and pass in a different classification value